tests/integration/it_reservations_test.py

```diff
@@ -1,42 +1,45 @@
-from datetime import datetime
+from http import HTTPStatus
 
 from flask import url_for
 
 import pytest
-from timeless.restaurants.models import Reservation, ReservationStatus
-
-def test_create_reservation(client, db_session):
-    reservation = Reservation.query.get(100)
-    if reservation:
-        db_session.delete(reservation)
-        db_session.commit()
-
-    comment = "Comment for verification"
-    response = client.post(url_for("reservations.create"), data={
-        "id": 100,
-        "start_time": datetime(2019, 2, 20, 21, 30),
-        "end_time": datetime(2019, 2, 20, 22, 30),
-        "customer_id": 1,
-        "num_of_persons": 2,
-        "comment": comment,
-        "status": ReservationStatus.confirmed.name
-    })
-    assert response.location.endswith(url_for("reservations.list_reservations"))
+from tests import factories
+from timeless.restaurants.models import Reservation
+
+
+@pytest.mark.skip
+def test_list(client):
+    reservation = factories.ReservationFactory()
+    response = client.get(url_for("reservations.list"))
+    assert response.status_code == HTTPStatus.OK
+    assert reservation.comment in response.data
+
+
+def test_create(client):
+    reservation = factories.ReservationFactory.build()
+    reservation_dict = {
+        column.name: str(getattr(reservation, column.name))
+        for column in reservation.__table__.columns
+    }
```

Comment:
@vladarefiev I've no idea what the code above is doing, is there some simpler/more understandable way of writing the same logic?

Reply:
@amihaiemil I've just added a comment. Hope it will be helpful for team.